src/routes/(app)/admin/email-alias/emailutils.ts

```diff
@@ -26,12 +26,19 @@ export const fuseEmail = (email: Email): string => {
 };
 
 const emailRegex =
+  /^[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
```

Comment:
Could be shortened to this I think, please double check
```suggestion
  /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
```

Reply:
Fixed